src/components/Endorsements.astro

```diff
@@ -0,0 +1,56 @@
+---
+// © 2024 Vlad-Stefan Harbuz <vlad@vladh.net>
+// SPDX-License-Identifier: Apache-2.0
+
+import Endorsement from "./Endorsement.astro";
+---
+
+<div class="endorsement-group">
+  <Endorsement
+    name="Johannes Schickling"
+    handle="@schickling"
+    imageUrl="/images/endorsers/schickling.webp"
+    sourceUrl="https://x.com/schickling/status/1832361798130954335"
+  >
+    <p>I can't overstate how excited I am about this. ❤️</p>
+  </Endorsement>
+  <Endorsement
+    name="Raymmar"
+    handle="@raymmar_"
+    imageUrl="/images/endorsers/raymmar.webp"
+    sourceUrl="https://x.com/raymmar_/status/1832464000996548806"
+  >
+    <p>Great initiative. [Vital] for the next step of internet growth. 💪</p>
+  </Endorsement>
+  <Endorsement
+    name="Xavier Damman"
+    handle="@xdamman"
+    imageUrl="/images/endorsers/xdamman.webp"
+    sourceUrl="https://x.com/xdamman/status/1837021472499343492"
+  >
+    <p
+      >Great initiative to get companies who wouldn’t exist without the digital public good of open source software to
+      give back.</p
+    >
+  </Endorsement>
+</div>
+
+<style>
+</style>
+
+<script>
+  import { shuffle, range } from "../util.ts";
+
+  const N_TO_CHOOSE = 3;
```

Comment:
Can we drop to one on homepage and make a page at `/endorsements` with all posts? "EXPLORE ALL ENDORSEMENTS" link similar to "EXPLORE ALL COMPANIES"